Rakefile

```diff
@@ -1,6 +1,20 @@
+# frozen_string_literal: true
+
 # Add your own tasks in files placed in lib/tasks ending in .rake,
 # for example lib/tasks/capistrano.rake, and they will automatically be available to Rake.
 
-require_relative "config/application"
+require_relative 'config/application'
+
+begin
+  require 'rspec/core/rake_task'
+
+  desc 'Run all examples'
+
+  RSpec::Core::RakeTask.new(:spec) do |t|
+    t.rspec_opts = %w[--color]
+    t.pattern = 'spec/**/*_spec.rb'
+  end
+rescue LoadError
+end
```

Comment:
What is that?
Remove it

Reply:
> What is that?
> Remove it

this settings to fix this

![image](https://user-images.githubusercontent.com/11984862/114295817-3f308c80-9ab0-11eb-822f-2aca2e411fe6.png)